aws-devicefarm-gradle-plugin/src/main/groovy/com.amazonaws.devicefarm/extension/DeviceFarmExtension.groovy

```diff
@@ -42,6 +42,12 @@ class DeviceFarmExtension {
      */
     String endpointOverride
 
+    /**
+     * [Optional] Wait for DeviceFarm to finish running the tests and consume results
+     * Default: False
+     */
+    boolean wait = false
```

Comment:
I believe that this needs to be renamed as gradle fails with the message:

> A problem occurred evaluating project ':app'.
\> Failed to apply plugin [id 'devicefarm']
   \> Could not create an instance of type com.amazonaws.devicefarm.extension.DeviceFarmExtension.
      \> Could not generate a decorated class for type DeviceFarmExtension.
         \> class com.amazonaws.devicefarm.extension.DeviceFarmExtension_Decorated overrides final method wait.(J)V